### ISSUE TITLE
Is it possible to export multiple tables?

### ISSUE THREAD

**Issue by andreElrico** [HalitTalha/mat-table-extensions#18]
Is it possible to have like a dialog to chose "m" von "n" mat-tables and export those "m" different table data into **1** xlsx with m **named** sheets? How could we construct something like this?

**Comment by HalitTalha** [comment]
Unfortunately currently it is not possible. I'll label this as a feature request.
I assume we can provide this feature sometime in the future through a service. Thanks for communicating.

**Comment by pantonis** [comment]
I managed to do that by calling each exporter

```
<button type="button" class="buttonExport"
                                (click)="exporter.exportTable('csv', {fileName:'Table 1'});
exporterValues.exportTable('csv', {fileName:'Table 2'})">
                                <i class="fas fa-file-excel"></i>
                                Export
                            </button>
```

**Comment by znotdead** [comment]
> @andreElrico I managed to do that by calling each exporter
> 
> ```
> <button type="button" class="buttonExport" (click)="table1Exporter.exportTable('csv', {fileName:'Table 1'});table2Exporter.exportTable('csv', {fileName:'Table 2'})">
> 	<i class="fas fa-file-excel"></i>
> 	Export
> </button>
> ```

It looks like it's not one file in XLSX format with 2 sheets you are getting in the end.

**Comment by chriszrc** [comment]
@andreElrico Yes, this is what I ended up using as well. But in my situation, sometimes I'm using multiple tables to display information in a custom format, but I'd like to export as a single sheet. Any way to combine the datasources perhaps?
